python/singa/image_tool.py

```diff
@@ -120,14 +121,26 @@ def resize(img, small_size):
     '''Resize the image to make the smaller side be at the given size'''
     size = img.size
     if size[0] < size[1]:
-        new_size = (small_size, int(small_size*size[1]/size[0]))
+        new_size = (small_size, int(small_size * size[1] / size[0]))
     else:
-        new_size = (int(small_size*size[0]/size[1]), small_size)
+        new_size = (int(small_size * size[0] / size[1]), small_size)
     new_img = img.resize(new_size)
     # print 'resize to (%d,%d)' % new_size
     return new_img
 
 
+def scale(img, small_size):
+    '''Scale the image to make the smaller side be at the give size
+        This method is to replace original resize'''
+    return resize(img, small_size)
+
+
+def resize_by_hw(img, (width, height)):
```

Comment:
better change the order of width and height to be consistent with the name 'hw'.